prow/plugins/releasenote/releasenote.go

```diff
@@ -81,11 +84,12 @@ func handle(gc githubClient, log *logrus.Entry, ic github.IssueCommentEvent) err
 		return nil
 	}
 
-	// Only allow authors and assignees to add labels.
-	isAssignee := ic.Issue.IsAssignee(ic.Comment.User.Login)
+	// Only allow authors and org members to add labels.
+	isMember, _ := gc.IsMember(trustedOrg, ic.Comment.User.Login)
```

Comment:
Just propagate up the error. Errors in `IsMember` are extremely rare and indicate that something catastrophic is going on, such as GitHub is down. In this case there's no need to continue.